<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeachingBox</class>
 <widget class="QWidget" name="TeachingBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TeachingBox</string>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>400</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>540</width>
     <height>400</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>540</width>
     <height>400</height>
    </size>
   </property>
   <widget class="QGroupBox" name="TaskGpBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>341</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>TaskManager</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="MvPtrnLb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>55</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Moving Pattrens:</string>
     </property>
    </widget>
    <widget class="QLabel" name="AvpotsLb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Available Points:</string>
     </property>
    </widget>
    <widget class="QLabel" name="CordsLb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Coordinates:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="MvModeCbox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>MOVJ</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MOVL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MOVC</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="AvpotsCbox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="CordsCbox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>130</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>切换机器人运动时坐标系</string>
     </property>
     <item>
      <property name="text">
       <string>Joint</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Rectg</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tool</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>User</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="crtBtn">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>创建机器人作业文件</string>
     </property>
     <property name="text">
      <string>Create </string>
     </property>
    </widget>
    <widget class="QPushButton" name="strBtn">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>开始机器人作业</string>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stpBtn">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>停止当前机器人作业</string>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rewBtn">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>100</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;回读机器人上次作业指令&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Review</string>
     </property>
    </widget>
    <widget class="QLabel" name="MvspedLb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Moving Speed:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="MvsdSpBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="value">
      <double>50.000000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="getPsBtn">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>130</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>获取当前末端点</string>
     </property>
     <property name="text">
      <string>Get Current Points</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="ModeGpBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string>Teaching Patterns</string>
     </property>
     <widget class="QRadioButton" name="JtRdBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Joint</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="WdRdBtn">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Rectg</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="ToRdBtn">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Tool</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="UrRdBtn">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>User</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>331</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Teaching Control</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QPushButton" name="Xplus">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="XaxLb">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>40</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>  J1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Xsub">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="Yplus">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="YaxLb">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>20</y>
        <width>40</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>  J2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Ysub">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="Zplus">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="ZaxLb">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>40</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>  J3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Zsub">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="RXplus">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>50</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="RXaxLb">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>50</y>
        <width>40</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>  J4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="RXsub">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>50</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="RYplus">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="RYaxLb">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>40</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>  J5</string>
      </property>
     </widget>
     <widget class="QPushButton" name="RYsub">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>80</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="RZplus">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>80</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="RZaxLb">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>80</y>
        <width>40</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>  J6</string>
      </property>
     </widget>
     <widget class="QPushButton" name="RZsub">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>80</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="title">
     <string>Running Status</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="ContStLb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>55</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect status:</string>
     </property>
    </widget>
    <widget class="QToolButton" name="ContRdBtn">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>55</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>连接到机器人</string>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
    <widget class="QLabel" name="SvmtStLb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Servo motor:</string>
     </property>
    </widget>
    <widget class="QToolButton" name="SvmtRdBtn">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>给伺服电机上电</string>
     </property>
     <property name="text">
      <string>UnWorking</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="HostInEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>机器人IP地址</string>
     </property>
     <property name="text">
      <string>192.168.58.110</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Host IP:</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>190</width>
      <height>210</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="autoScrollMargin">
     <number>20</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>85</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>20</number>
    </attribute>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>4</width>
     <height>12</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
